src/open-url.coffee

```diff
@@ -14,14 +14,12 @@ firebaseUrl = process.env.HUBOT_FIREBASE_URL
 firebaseAuth = process.env.HUBOT_FIREBASE_SECRET
 
 bookmarks = {}
-confirmations = [
-	'Done.'
-]
-holdings = [
-	'Doing'
-]
+
+Personality = require('./personality.coffee')
+Promise = require 'bluebird'
```

Comment:
It's completely preference but I find it's more obvious if the external dependencies (in this case `bluebird`) are imported before the local dependencies, and this is a standard that others do follow, but it's not a hard and fast rule.